include/nbl/video/COpenGL_Swapchain.h

```diff
@@ -11,6 +11,7 @@
 #include "nbl/video/COpenGLFence.h"
 #include "nbl/video/COpenGLSemaphore.h"
 #include "nbl/video/COpenGLImage.h"
+#include "nbl/video/COpenGL_Queue.h"
```

Comment:
see how many of these you can remove if you move the implementation to .cpp file and forward declare the classes